### PR TITLE
add trendyol company logo

### PR DESCRIPTION
**Pull request by previousdeveloper**
Overview

We were established in 2010 with an ambition to provide our customers and sellers a flawless e-commerce experience. Today we are not only Turkey's leading e-commerce company but also amongst the world's leading platforms.
By supporting the digitization of our tradesmen and SMEs, we allow them to reach more customers and grow their business. Also, we support women's participation in the digital economy, domestic production, and local producers. Thus, we are able to make a significant contribution to the growth of our country's economy and increase employment.
As Turkey's Trendyol, we maintain our country's leading R&D center, Trendyol Tech, the most rapidly developing transport network, Trendyol Express, fast delivery service for food and grocery, Trendyol Go, and the largest second-hand product platform, Dolap, with an aim to grow continuously whilst driving positive impact.

Istio is being heavily used to facilitate service communication, security, monitoring, and distributed tracing. 
Some production learnings here,

https://medium.com/trendyol-tech/service-mesh-journey-at-trendyol-part-1-468033817d44
https://medium.com/trendyol-tech/service-mesh-journey-at-trendyol-part-2-9606f9953c1
https://medium.com/trendyol-tech/istio-mtls-between-multiple-clusters-b146829e40aa

Proposal
We would like to be part of Istio's list of customers.

Our organisation
https://github.com/Trendyol

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by previousdeveloper** [comment]
@googlebot I signed it!

**Comment by craigbox** [comment]
Nice!  Is this you?  https://twitter.com/mstrYoda_/status/1364837000025690114

This will be a first for me:
/cherrypick release-1.10

File: data/companies.yml [hunk 1]
```diff
@@ -434,3 +434,6 @@ users:
   - name: "Zendesk"
     logo: "/logos/zendesk.svg"
     url: "https://www.zendesk.com/"
+  - name: "Trendyol"
+    logo: "/logos/trendyol.png"
+    url: "https://www.trendyol.com/"
```
Comment (craigbox) [inline data/companies.yml:439]: Note to self: I should update the code to sort the names alphabetically
Note to you: could you please revise the PR to sort your name alphabetically? 🙂

**Review by craigbox — CHANGES_REQUESTED** [review]

File: data/companies.yml (hunk 1 above)
Comment (previousdeveloper) [reply data/companies.yml]: Thanks, I changed it can you check? :slightly_smiling_face:

**Comment by previousdeveloper** [comment]
> Nice! Is this you? https://twitter.com/mstrYoda_/status/1364837000025690114
> 
> This will be a first for me:
> /cherrypick release-1.10

He is my colleague

**Comment by craigbox** [comment]
Thank you both for using Istio 🙏

>

**Review by craigbox — APPROVED** [review]

**Pull request merged**